geostore/tasks.py

```diff
@@ -22,3 +30,45 @@ def layer_relations_set_destinations(relation_id):
         feature_update_relations_destinations.delay(feature_id, relation_id)
 
     return True
+
+
+def send_mail_generated_file(user, path):
+    context = {"username": user.username, "file": path}
+    html = get_template('email_async.html')
+    html_content = html.render(context)
+    send_mail(_('Your file is ready'), html_content, "coucou@coucou.coucou", [user.email], fail_silently=False)
```

Comment:
```suggestion
    send_mail(_('Your file is ready'), html_content, "coucou@coucou.coucou", [user.email], fail_silently=True)
```

il faut pas se prendre une exception si le destinataire n'existe pas ou autre probleme